platform/android/app/src/main/kotlin/coop/polypoly/polypod/settings/MainFragment.kt

```diff
@@ -37,4 +52,14 @@ class MainFragment : PreferenceFragmentCompat() {
                 true
             }
     }
+
+    private fun onAuthRequest(newStatus: Boolean, onReturn: (Boolean) -> Unit) {
+        Authentication.setUp(
+            requireActivity(),
+            newStatus
+        ) { success ->
+            onReturn(success)
```

Comment:
Related to https://github.com/polypoly-eu/polyPod/pull/854/files#r866797016, I see that here you use the value returned in callback, but then on line `24` I don't see it used. The `isBiometricEnabled` is retrieved from Preferences. Am I missing something?

Reply:
so we mainly do:
```
val pref = findPreference<SwitchPreference>("biometricEnabledKey")
pref?.isChecked =  Preferences.isBiometricEnabled(requireContext())
```
in order to update the UI according to the value of Preferences rather than to depend on the value that callback returns as it's async

Reply:
So therefore, the callback should probably be `() -> Unit` instead of `(Boolean) -> Boolean`